pkg/api/kptfile/v1/validation.go

```diff
@@ -106,14 +106,16 @@ func (f *Function) validate(fnType string, idx int, pkgPath types.UniquePath) er
 //	path-component                  := alpha-numeric [separator alpha-numeric]*
 // 	alpha-numeric                   := /[a-z0-9]+/
 //	separator                       := /[_.]|__|[-]*/
-func validateFunctionName(name string) error {
+func ValidateFunctionName(name string) error {
```

Comment:
Can we rename this to `ValidateFunctionImageURL` ?

Reply:
done